Sources/CorePayments/Networking/GraphQL/GraphQLHTTPResponse.swift

```diff
@@ -1,5 +1,5 @@
-/// Used to decode the HTTP reponse body of GraphQL requests
-struct GraphQLHTTPResponse<T: Codable>: Codable {
+/// :nodoc: Used to decode the HTTP reponse body of GraphQL requests
```

Comment:
Ticket for this work - DTPPCPSDK-726

Reply:
Yeah - it would be a good test for structure/format since we don't have anything currently. We can start fresh with DocC vs switching from Jazzy down the road. 🎉